napps/kytos/of_l2ls/main.py

```diff
@@ -51,6 +52,9 @@ def handle_packet_in(self, event):
 
         switch.update_mac_table(ethernet.source, in_port)
 
+        if packet_in.reason != PacketInReason.OFPR_NO_MATCH:
+            return
```

Comment:
In l2ls, it is flood or flow_mod.
If someone desires a different behavior, then l2ls should be disabled. As long as it's enabled, it floods or flow_mods.